fred2/orienteditor.cpp

```diff
@@ -182,9 +182,27 @@ BOOL orient_editor::OnInitDialog()
 	return TRUE;
 }
 
-bool orient_editor::close(float val, const CString &str)
+bool orient_editor::close(float val, const CString &input_str)
 {
-	float diff = val - convert(str);
+	CString str;
+	str.Format("%.01f", val);
+	val = convert(str);
+
+	float input_val = convert(input_str);
+
+	float diff = val - input_val;
+	return diff > -input_threshold && diff < input_threshold;
+}
+
+bool orient_editor::angle_close(float rad, const CString &input_str)
+{
+	CString str;
+	str.Format("%.01f", to_degrees(rad));
+	float deg = convert(str);
+
+	float input_deg = normalize_degrees(convert(input_str));
```

Comment:
Why the different calls to `to_degrees()` vs. `normalize_degrees()`?

Reply:
One is converting units and then normalizing; the other is just normalizing.